src/main/resources/application.yml

```diff
@@ -113,6 +113,7 @@ community-api:
     ended-referral: "/secure/offenders/crn/{crn}/referral/end/context/{contextName}"
     book-appointment: "/secure/offenders/crn/{crn}/sentence/{sentenceId}/appointments/context/{contextName}"
     reschedule-appointment: "/secure/offenders/crn/{crn}/appointments/{appointmentId}/reschedule/context/{contextName}"
+    relocate-appointment: "/secure/offenders/crn/{crn}/appointments/{appointmentId}/relocate/context/{contextName}"
```

Comment:
A context is not required in this situation as there is no mapping required that is specific to Interventions (aka CRS aka R&M)

Reply:
Thanks, fixed.